src/OpenTelemetry.Instrumentation.Hangfire/Implementation/HangfireInstrumentationJobFilterAttribute.cs

```diff
@@ -30,8 +36,16 @@ public void OnPerforming(PerformingContext performingContext)
                 return;
             }
 
+            var activityContextData = performingContext.GetJobParameter<Dictionary<string, string>>(HangfireInstrumentationConstants.ActivityContextKey);
+            ActivityContext parentContext = default;
+            if (activityContextData is not null)
+            {
+                var propagationContext = Propagator.Extract(new PropagationContext(default, Baggage.Current), activityContextData, ExtractActivityProperties);
```

Comment:
`Propagators.DefaultTextMapPropagator` can change a runtime. I don't think you want to hold on to it in a field on this class and instead always call `Propagators.DefaultTextMapPropagator.Extract` here. Likewise for where you're calling `Inject`.

Reply:
done